### DISCUSSION TITLE
if i use Cef.DoMessageLoopWork() sometime this code throw OutOfMemExcetion

### DISCUSSION THREAD

**Discussion by PingXD** [cefsharp/CefSharp#3836] [General]
![image](https://user-images.githubusercontent.com/48038690/135374017-e1797dca-c90b-4fed-b1c4-638cd4a35b4a.png)
version 86.0.241.0

**Comment by PingXD** [comment]
![image](https://user-images.githubusercontent.com/48038690/135375959-822701e7-b959-441b-9ccd-d88537c21bef.png)

**Comment by amaitland** [comment]
> version 86.0.241.0

This is an old unsupported version. Please test with version `94`.

If you are using a `x86` process make sure it is large address aware as per https://github.com/cefsharp/CefSharp/wiki/General-Usage#win32-out-of-memory.

There are potentially many reasons your process has run out of memory, just because `CefSharp` is throwing the exception doesn't mean it's the cause, just means it attempted to allocate memory and there wasn't enough free.

No images of code/exceptions

In future please use discussions to ask questions/support. https://github.com/cefsharp/CefSharp/discussions

**Reply by PingXD** [reply]
The memory is only 400mb😖

**Reply by amaitland** [reply]
The exception is clear, your process has insufficient memory to allocate the `byte[]`.

I'd suggest using [Process Monitor](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer) to check for memory.

If the problem reproduces in version `94` then you'll need to provide a detailed code example. As it is all I know is you've run out of memory.

**Reply by amaitland** [reply]
Please take the time to read over https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support
